app/src/main/res/layout/quicksettings_website_info.xml

```diff
@@ -35,24 +35,25 @@
             style="@style/QuickSettingsText"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
+            android:paddingStart="8dp"
             android:paddingTop="8dp"
-            tools:text="https://wikipedia.org" />
+            tools:text="https://wikipedia.org"
+            tools:ignore="RtlSymmetry" />
```

Comment:
Same concern as above, let's use paddingHorizontal instead of paddingStart to specify some paddingEnd

Reply:
See https://github.com/mozilla-mobile/fenix/pull/20144#discussion_r668196582